src/v0/destinations/rakuten/networkHandler.js

```diff
@@ -0,0 +1,102 @@
+const { NetworkError } = require('@rudderstack/integrations-lib');
+const { httpSend } = require('../../../adapters/network');
+const {
+  processAxiosResponse,
+  getDynamicErrorType,
+} = require('../../../adapters/utils/networkUtils');
+const { TAG_NAMES } = require('../../util/tags');
+const { HTTP_STATUS_CODES } = require('../../util/constant');
+
+const DESTINATION = 'RAKUTEN';
+const prepareProxyRequest = (request) => request;
+const proxyRequest = async (request, destType) => {
+  const { endpoint, data, method, params, headers } = prepareProxyRequest(request);
+  const requestOptions = {
+    url: endpoint,
+    data,
+    params,
+    headers,
+    method,
+  };
+  const response = await httpSend(requestOptions, { feature: 'proxy', destType });
+  return response;
+};
+const extractContent = (xmlPayload, tagName) => {
+  const pattern = new RegExp(`<${tagName}>(.*?)</${tagName}>`);
+  const match = xmlPayload.match(pattern);
+  return match ? match[1] : null;
+};
+
+const responseHandler = (destinationResponse) => {
```

Comment:
we can write unit tests for this also

Reply:
Proxy already handles it and it does not seem complex response Handler to write unite test cases for.
WDYT?

Reply:
I know that but such things are better handled at unit tests and it is easy to test with unit tests

Reply:
Done